docs/understanding-tracking-design/json-schema-type-casting-rules/index.md

```diff
@@ -0,0 +1,370 @@
+---
+title: "Type casting rules for json schemas"
+date: "2021-12-30"
+sidebar_position: 100
+hide_table_of_contents: true
+---
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+_The row order in this table is important.  Type lookup stops after first match is found scanning from top to bottom (with the single exception of "null" - first row in the table)_
+<Tabs groupId="type-casting">
+  <TabItem value="redshift" label="Redshift" default>
+
+<table>
+<thead>
+<td>Json Schema</td>
+<td>Redshift Type</td>
+<td>Notes</td>
+</thead>
+<tbody>
+<tr>
+<td >
+<pre lang="json">&#123;<br/>
```

Comment:
I recommend using this instead of `pre`: https://docusaurus.io/docs/markdown-features/code-blocks#usage-in-jsx. I think it will have better highlighting